### ISSUE TITLE
Setting validation drop list from one sheet to another

### ISSUE THREAD

**Issue by dethlex** [qax-os/excelize#1019]
**Description**
I want to set up data validation drop list from one sheet to another, but property Formula1 in DataValidation struct not fills correctly in method SetSqrefDropList, so drop down list doesn't work properly.

For example:
```go
dvRange := excelize.NewDataValidation(true)
dvRange.ShowErrorMessage = true
dvRange.Sqref = "A1:A1"
dvRange.SetSqrefDropList("Sheet2!$A$1:$A$3" true)
f.AddDataValidation("Sheet1", dvRange);
```

Possibly, instead of just setting variable
_datavalidation.go:157_ : `dd.Formula1 = sqref`
we may use special formatting
_datavalidation.go:157_ : `dd.Formula1 = fmt.Sprintf("<formula1>%s</formula1>", sqref)`

After this changes drop down list works normally.

**Output of `go version`:**

```text
go version go1.16.4 linux/amd64
```

**Excelize version or commit ID:**

```text
v2.4.1
```

**Environment details (OS, Microsoft Excel™ version, physical, etc.):**
OS: Linux 5.11.0-34-generic #36~20.04.1-Ubuntu SMP Fri Aug 27 08:06:32 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
Office: LibreOffice Calc 6.4.7.2

**Comment by xuri** [comment]
Thanks for your issue, there is an incorrect absolute cell reference expression in your code, please use`"Sheet2!$A$1:$A$3"` instead of `"Sheet2!A$1$:A$3$"`, it will work correctly.

**Comment by dethlex** [comment]
Excuse me, I've made a mistake in this example, because I'm calculating adresses with _excelize.CoordinatesToCellName_ and wrote there by memory :)
So, problem still exists..

**Comment by xuri** [comment]
Could you provide a sample code to reproduce this issue?

**Comment by xuri** [comment]
This issue seems duplate with #979, it has been fixed in commit [`cf9fbaf`](https://github.com/qax-os/excelize/commit/cf9fbafdd805874267a0f5d27fd1c720b148ec91),  please try to use the master branch code.

**Comment by xuri** [comment]
This issue seems duplicate with #986, it has been fixed in commit [`cf9fbaf`](https://github.com/qax-os/excelize/commit/43a057b1eaeb810495618d70c2dc2d3e5df1fea4),  please try to use the master branch code.

**Comment by xuri** [comment]
This issue seems duplicate with #986, it has been fixed in commit [`43a057b`](https://github.com/qax-os/excelize/commit/43a057b1eaeb810495618d70c2dc2d3e5df1fea4),  please try to use the master branch code.

**Comment by dethlex** [comment]
Ok, sorry for duplicate, thank you!
